plugins/apollo-explorer/api-report.md

```diff
@@ -0,0 +1,52 @@
+## API Report File for "@backstage/plugin-apollo-explorer"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+/// <reference types="react" />
+
+import { BackstagePlugin } from '@backstage/core-plugin-api';
+import { JSONObject } from '@apollo/explorer/src/helpers/types';
+import { RouteRef } from '@backstage/core-plugin-api';
+
+// Warning: (ae-missing-release-tag) "ApolloExplorerPage" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

Comment:
Think you need to add a `tsdoc` `@public` tag to the components that you're exporting.